### ISSUE TITLE
standard v.7.4.2 number_format function is wrongly documented

### ISSUE THREAD

**Issue by rimace** [eclipse/pdt#129]
**Describe the bug**
The stanrdard function (in the standard.php file) "number_format" is not correctly documented.

**Describe the eclipse environment**
A clear and concise description about Eclipse Version 2021-12, PDT version 8.0.

**Describe your system**
 - OS: Fedora Linux
 - Version 35
 - PHP Version 7.4
 - Utilis Eclipse IDE

**To Reproduce**
1. Look at the standard classes/functions of php. 
2. See that number_format() function has only 2 arguments (but it should be 4)

**Expected behavior**
The function number_format should be documented as:
```
 number_format(
    float $num,
    int $decimals = 0,
    ?string $decimal_separator = ".",
    ?string $thousands_separator = ","
): string

```
**Additional context**

**Comment by agmckee** [comment]
Looks like the language documentation may have been generated with an out-of-date version of the phpdoc repo. There was a note in the [commit](https://github.com/eclipse/pdt/commit/1cf9b8ff190d85587b76a89efd1f323cd9996522) about updating the documentation once the PHP 7.4 binaries were available.

**Comment by zulus** [comment]
Our language generate need some enhancements. For now it incorrectly "parse" manuala references. I made some fixes for #148, but it still need improvements.

**Comment by zulus** [comment]
I switched from tons of "preg_replace" to DOMDocument (after compile manual) + some smaller hacks. From my point of view language library is a lot better now
